creator-node/tsconfig.json

```
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./src",
```

```suggestion
    "baseUrl": "./src",
    "pretty": true,
```
[suggestion](https://www.typescriptlang.org/tsconfig/#pretty)

we also probably want to keep [sourcemaps](https://www.typescriptlang.org/tsconfig/#sourceMap) as it will facilitate debugging.

---

i will add this